src/main/java/com/icthh/xm/ms/timeline/config/metrics/JHipsterHealthIndicatorConfiguration.java

```diff
@@ -1,10 +1,15 @@
 package com.icthh.xm.ms.timeline.config.metrics;
+
+import static com.icthh.xm.ms.timeline.config.Constants.CASSANDRA_IMPL;
+
 import com.datastax.driver.core.Session;
 import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+@ConditionalOnProperty(name = "application.timeline-service-impl", havingValue = CASSANDRA_IMPL)
```

Comment:
Why we need this condition for this bean?

Reply:
This health check for cassandra
wrong class name
renamed